code/modules/organs/external/_external.dm

```diff
@@ -325,6 +331,15 @@
 
 	return all_items
 
+/obj/item/organ/external/proc/get_limbs_recursive(var/no_stumps = FALSE)
+	var/list/all_limbs = list()
+	for(var/obj/item/organ/external/child in children)
+		if(no_stumps && child.is_stump())
+			continue
+		all_limbs += child
+		all_limbs.Add(child.get_limbs_recursive(no_stumps))
```

Comment:
This is going to insert nulls if nothing is returned from the recursive limb proc.

Reply:
It inserts nulls if the list passed to Add is empty?

Reply:
Tested it on wrench and it does not insert nulls at all?